tests/unit/iontests.js

```diff
@@ -51,15 +50,31 @@ define([
       'good/utf32.ion',
     ];
 
+    // For debugging, put single files in this list to have the test run only
+    // that file.   (But don't forget to clean this up on checkin!)
+    var debugList = [
```

Comment:
https://theintern.github.io/intern/#common-config

Have not really used it myself but given a naming convention for the tests would the `grep` option suffice here?